### PR TITLE
feat: add animation-clip doc

### PR DESCRIPTION
**Pull request by zhuxudong**

### CODE REVIEW

File: docs/editor-animation-clip.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+- 举例，新建了一个名为 **animation_copy **的动画切片，然后只保留最后 5 帧动画：
+
+![image.png](https://cdn.nlark.com/yuque/0/2021/png/381718/1622617643472-17314b46-06a6-4368-952a-416814227566.png#clientId=u8b862108-a194-4&from=paste&height=132&id=hPacq&margin=%5Bobject%20Object%5D&name=image.png&originHeight=395&originWidth=1464&originalType=binary&size=58042&status=done&style=none&taskId=uaf76c052-c2dc-420c-a43e-01bcf153cfe&width=488)
+![image.png](https://cdn.nlark.com/yuque/0/2021/png/381718/1622617795573-fb73aeec-fbb0-4851-9f8a-1a1909b789d8.png#clientId=u8b862108-a194-4&from=paste&height=256&id=C8pN7&margin=%5Bobject%20Object%5D&name=image.png&originHeight=769&originWidth=1412&originalType=binary&size=170314&status=done&style=none&taskId=u4736c5bf-33f8-4053-bba9-fd81c9e343b&width=470.6666666666667)![image.png](https://cdn.nlark.com/yuque/0/2021/png/381718/1622617813768-69bf92bc-ec55-4b00-9ff8-b7ce324e9526.png#clientId=u8b862108-a194-4&from=paste&height=157&id=CFO2b&margin=%5Bobject%20Object%5D&name=image.png&originHeight=471&originWidth=1483&originalType=binary&size=62305&status=done&style=none&taskId=u78d4ba08-ab6b-45d2-b5cd-da55a8c9280&width=494.3333333333333)
```
Comment (cptbtptpbcptdtptp) [inline docs/editor-animation-clip.zh-CN.md:29]: Missing line break

**Review by cptbtptpbcptdtptp — COMMENTED** [review]
Check the format

**Review by eyworldwide — APPROVED** [review]
+1

File: docs/editor-animation-clip.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+为减少沟通成本，本文列举了几种常见的动画切片方法，导出 GLTF 方便 Oasis 引擎直接使用，也可以通过 [GLTF 预览](https://oasisengine.cn/gltf-viewer) 页面进行功能校验。
+​
+
+Blender 的动画编辑页面非常友好，能够清晰地可视化显示受动画影响的节点，并且在时间轴上显示关键帧，因此推荐使用 Blender 进行动画切片。
+​
+
+### Blender
+
+- 打开 Blender，导入 Blender 支持的模型格式，然后切换到 **动画编辑** 窗口：
```
Comment (eyworldwide) [inline docs/editor-animation-clip.zh-CN.md:18]: Use ordered list instead，step by step.

File: docs/editor-animation-clip.zh-CN.md (hunk 1 above)
Comment (zhuxudong) [reply docs/editor-animation-clip.zh-CN.md:29]: done

File: docs/editor-animation-clip.zh-CN.md (hunk 2 above)
Comment (zhuxudong) [reply docs/editor-animation-clip.zh-CN.md:18]: done

**Pull request merged**